tests/test_transforms/test_timestamp/test_holiday_transform.py

```diff
@@ -201,11 +286,22 @@ def test_holidays_min(iso_code: str, answer: np.array, two_segments_simple_ts_mi
         (pd.date_range(start="2019-11-25", end="2021-02-25", freq="M")),
     ),
 )
-def test_holidays_failed(index: pd.DatetimeIndex, two_segments_simple_ts_daily: TSDataset):
-    ts = two_segments_simple_ts_daily
+def test_holidays_failed(index: pd.DatetimeIndex, two_segments_simple_ts_day_15min: TSDataset):
+    ts = two_segments_simple_ts_day_15min
     ts.df.index = index
     holidays_finder = HolidayTransform(out_column="holiday")
-    with pytest.raises(ValueError, match="Frequency of data should be no more than daily."):
+    with pytest.raises(
+        ValueError, match="For binary and category modes frequency of data should be no more than daily."
+    ):
+        ts = holidays_finder.fit_transform(ts)
+
+
+def test_holidays_days_count_mode_failed(two_segments_simple_ts_daily: TSDataset):
```

Comment:
We should add here case for `two_segments_simple_ts_hour`, by using `pytest.param` like in the test above.